qemu/tests/balloon_hotplug.py

```diff
@@ -66,6 +83,11 @@ def run_pm_test(pm_test, plug_type):
             err += ("\nDevice is not in qtree %ss after hotplug:\n%s"
                     % (pause, vm.monitor.info("qtree")))
 
+        if params.get("blnsrv_enable", "no") == "yes":
```

Comment:
Can add a comment here that balloon service check only run in windows guest

Reply:
I will add this info in followed error_context.context.
Thanks